### ISSUE TITLE
[BUG] The installer give incorrect minimum disk requirement

### ISSUE THREAD

**Issue by elct9620** [harvester/harvester#2051]
**Describe the bug**

When installing Harvester, the installer will allow >= 60G disk to process the next step.
But the Prometheus requires more than 60G volume for monitoring and always shows the monitor is 502 error.

**To Reproduce**

1. When installing Harvester, use a small disk (test it in Virtual Machine)

**Expected behavior**

It should block until use the minimum disk is satiety.

**Support bundle**

**Environment:**
 - Harvester ISO version:  1.0.0
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630): ProxmoxVE 7.1

**Additional context**

**Comment by joshmeranda** [comment]
I can confirm the issues / errors you describe @elct9620, see the images below from a VM with 65 GB storage:

![graphs-no-load](https://user-images.githubusercontent.com/43455015/161319244-988cd8f8-ebf9-4997-a00a-fb2fd0a94dd5.png)

![bad-gateway](https://user-images.githubusercontent.com/43455015/161319286-d0b0bb29-254f-42dd-9b65-461cc4b84fb4.png)

However, I don't think it is necessarily the right choice to block until a minimum disk space is available. For one thing, the Prometheus monitoring is not vital to the functionality of Harvester as I am aware, and as seen in the image below the user is notified that they do not have the minimum recommended available storage space and given the choice to continue with the installation anyway:

![too-little-space](https://user-images.githubusercontent.com/43455015/161319309-fa7390ad-149c-48ef-8dc8-ba12b2bb33ff.png)

**Comment by joshmeranda** [comment]
We probably *could* be smarter with how we deal with this type of situation, or allow users to toggle non-vital functionality such as Prometheus and allow some version of a minimal harvester deployment.

**Comment by elct9620** [comment]
Agree, my concern is that behavior will let users confuse. I am trying to find the root cause of my dashboard getting errors. It wastes the time for the user but we cannot disable Prometheus if we didn't need it.

**Comment by elct9620** [comment]
Agree, my concern is that behavior will let users confuse. I am trying to find the root cause of my dashboard getting errors. It wastes the time for the user but we cannot disable Prometheus if we didn't need it.

**Comment by joshmeranda** [comment]
Sounds good, I'll open another issue to allow for disabling monitoring and link to this issue.
